### PR TITLE
TW edits, flow concept

### PR DESCRIPTION
**Pull request by keizobabybear**

### CODE REVIEW

File: docs/concepts/flows.md [hunk 1]
```diff
@@ -7,121 +7,100 @@ tags:
...
-
-However you run the flow, the Prefect API monitors the flow run, capturing flow run state for observability.
-
-When you run a flow that contains tasks or additional flows, Prefect will track the relationship of each child run to the parent flow run.
-
-![Prefect UI](/img/ui/timeline-flows.png)
+### Flow benefits
 
-## Writing flows
+Using Prefect flows offer several advantages:
```
Comment (billpalombi) [inline docs/concepts/flows.md:36]:
```suggestion
Using Prefect flows offers several advantages:
```

File: docs/concepts/flows.md [hunk 2]
```diff
@@ -7,121 +7,100 @@ tags:
...
+| `flow_run_name`                                    | Optional template string to define the name for flow runs. Can reference flow parameters or use a function. For example, you can use the `{name}-on-{date:%A}` value for dynamic flow run naming|
+| `task_runner`                                      | Optional task runner to use for task execution within the flow. Defaults to `ConcurrentTaskRunner`.|
+| `timeout_seconds`                                  | Optional maximum runtime (in seconds) for the flow. If exceeded, the flow is marked as failed.|
+| `validate_parameters`                              | Boolean indicating whether to validate flow parameters using Pydantic (`default: True`). Where possible, Prefect forces values into the correct type. For example, if you define a parameter as `x: int` and "5" is passed, it resolves to `5`.|
+| `version`                                          | Optional version string for the flow. Defaults to a hash of the file containing the wrapped function. If the file cannot be found, the `version` value is `null`. |
+
+### Examples
+The following examples show how to customize certain flow behaviors.
```
Comment (billpalombi) [inline docs/concepts/flows.md:101]: I interpret "behavior" to mean the way that my flow runs. While the task runner changes the flow's behavior by the definition, the description and flow_run_name are not so much behavior changes as they are record keeping changes.

File: docs/concepts/flows.md [hunk 3]
```diff
@@ -204,158 +181,81 @@ if __name__ == "__main__":
...
-    print(msg)
-    return msg
+You can trigger a flow run in several ways:
 
-@flow(name="Hello Flow")
-def hello_world(name="world"):
-    message = print_hello(name)
+* **Manually:** Calling the flow function directly in a script or interactive session.
+* **Scheduler:** Using external schedulers like `cron` to invoke the flow function periodically.
+* **Prefect Cloud/Server:** Creating [deployments](/concepts/deployments/) on Prefect Cloud or a locally run Prefect server. You can then schedule flow runs through the UI, API, or deployments.
```
Comment (billpalombi) [inline docs/concepts/flows.md:195]:
```suggestion
* **Prefect Cloud/Server:** Creating [deployments](/concepts/deployments/) on Prefect Cloud or a locally run Prefect server. You can then schedule flow runs through the UI, API, or SDK.
```

File: docs/concepts/flows.md [hunk 4]
```diff
@@ -204,158 +181,81 @@ if __name__ == "__main__":
...
-@flow(name="Hello Flow")
-def hello_world(name="world"):
-    message = print_hello(name)
-    message2 = print_hello_again(message)
+* **Conditional execution**: Subflows enable you to group tasks that only run under specific conditions. This simplifies conditional logic by allowing you to conditionally run the entire subflow instead of managing individual tasks.
 
-if __name__ == "__main__":
-    hello_world.visualize()
-```
+* **Parametrization**: Prefect's parameterization capabilities extend to subflows. This allows you to reuse the same group of tasks for different purposes by simply passing different parameters when invoking the subflow.
```
Comment (billpalombi) [inline docs/concepts/flows.md:226]: Throughout, sometimes the colon is included in the bolded text and sometimes not. Either is fine as long as its consistent.

File: docs/concepts/flows.md [hunk 5]
```diff
@@ -204,158 +181,81 @@ if __name__ == "__main__":
...
 
-!!! warning "Functions and code not inside of flows or tasks will still be run when calling `.visualize()`. This may have unintended consequences. Place your code into tasks to avoid unintended execution."
+Prefect tracks the state of each flow run, allowing you to monitor its progress and identify any issues. Here are the typical flow run states:
 
-!!! note "To use the `visualize()` method, Graphviz must be installed and on your PATH. Please install Graphviz from [http://www.graphviz.org/download/](http://www.graphviz.org/download/). And note: just installing the `graphviz` python package is not sufficient."
+* **Scheduled:** The flow run is waiting for execution based on a schedule.
+* **Running:** The flow and its tasks are actively executing.
+* **Failed:** The flow run encountered an error and did not complete successfully.
+* **Cancelled:** The flow run was manually stopped before completion.
+* **Succeeded:** The flow run finished execution without errors.
```
Comment (billpalombi) [inline docs/concepts/flows.md:216]: We do not have a Succeeded state.

File: docs/concepts/flows.md [hunk 6]
```diff
@@ -204,158 +181,81 @@ if __name__ == "__main__":
...
-Prefect cannot automatically produce a schematic for dynamic workflows, such as those with loops or if/else control flow.
-In this case, you can provide tasks with mock return values for use in the `visualize()` call.
+### Subflow execution
 
-```python
-from prefect import flow, task
-@task(viz_return_value=[4])
-def get_list():
-    return [1, 2, 3]
+Initiate a subflow run when you call a flow function inside another flow. The calling flow acts as the "parent" flow, while the created flow becomes the "child" or "subflow." The Prefect backend maintains a full visual representation of the subflow run, as if it were called directly.
```
Comment (billpalombi) [inline docs/concepts/flows.md:232]:
```suggestion
A subflow run is initiated when you call a flow function inside another flow run. The calling flow acts as the "parent" flow, while the created flow becomes the "child" or "subflow." Prefect maintains a full visual representation of the subflow run, as if it were called directly.
```

File: docs/concepts/flows.md [hunk 7]
```diff
@@ -497,296 +391,87 @@ It was Friday on 2021-01-01T02:00:19.180906
...
-22:22:37.060 | ERROR   | Flow run 'acrid-tuatara' - Encountered exception during execution:
-Traceback (most recent call last):...
-ValueError: This flow immediately fails
-```
-</div>
-
-### Return `none`
-
-A flow with no return statement is determined by the state of all of its task runs.
+This following code visualizes the below diagram:
```
Comment (billpalombi) [inline docs/concepts/flows.md:413]:
```suggestion
Visualizing the following code results in the below diagram:
```

**Review by billpalombi — COMMENTED** [review]
Quick pass through the first half.

File: docs/concepts/flows.md [hunk 8]
```diff
@@ -1103,66 +743,58 @@ async def greet_user():
...
+Upon successful validation, the flow run resumes, and the return value of the `pause_flow_run` is an instance of the `UserNameInput` model containing the provided data.
 
-For more in-depth information on receiving input from users when pausing and suspending flow runs, see the [Creating interactive workflows](/guides/creating-interactive-workflows/) guide.
+For more information on receiving input from users when pausing and suspending flow runs, see [Creating interactive workflows](/guides/creating-interactive-workflows/).
 
-## Canceling a flow run
+### Cancel flow runs
 
-You may cancel a scheduled or in-progress flow run from the CLI, UI, REST API, or Python client.
+Cancel a scheduled or in-progress flow run from the CLI, UI, REST API, or Python client to transition flow runs to a `Cancelling` state.
```
Comment (discdiver) [inline docs/concepts/flows.md:756]:
```suggestion
A scheduled or in-progress flow run that is the entrypoint flow for a deployment can be cancelled from the CLI, UI, REST API, or Python client.
```

File: docs/concepts/flows.md [hunk 9]
```diff
@@ -1103,66 +743,58 @@ async def greet_user():
...
 
-When cancellation is requested, the flow run is moved to a "Cancelling" state.
-If the deployment is a work pool-based deployemnt with a worker, then the worker monitors the state of flow runs and detects that cancellation has been requested.
-The worker then sends a signal to the flow run infrastructure, requesting termination of the run.
-If the run does not terminate after a grace period (default of 30 seconds), the infrastructure will be killed, ensuring the flow run exits.
+How it works:
```
Comment (discdiver) [inline docs/concepts/flows.md:758]: I would leave this section as is. I think the proposed changes don't capture the useful information. 

One change that would be good is saying that a deployment is required up front.

File: docs/concepts/flows.md [hunk 10]
```diff
@@ -1103,66 +743,58 @@ async def greet_user():
...
-    We are working on improving cases where cancellation can fail.
-    You can try the improved cancellation experience by enabling the `PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION` setting on your worker or agents:
+### Improved cancellation (experimental)
 
-    <div class="terminal">
-    ```bash
-    prefect config set PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION=True
-    ```
-    </div>
+Improve cancellation reliability by enabling the `PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION` setting on your worker or agents:
```
Comment (discdiver) [inline docs/concepts/flows.md:789]:
```suggestion
Improve cancellation reliability by enabling the `PREFECT_EXPERIMENTAL_ENABLE_ENHANCED_CANCELLATION` setting on your worker:
```

**Review by discdiver — COMMENTED** [review]
Nice work condensing the content! Please let me know if you have questions about suggested edits and comments.

File: docs/concepts/flows.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (billpalombi) [reply docs/concepts/flows.md:216]: +1

**Pull request closed without merging**